src/test/java/com/artipie/rpm/CliTest.java

```diff
@@ -48,4 +51,57 @@ void testWrongArgumentCount() {
             );
         }
     }
+
+    @Test
+    void testRunWithArgument(@TempDir final Path temp) {
+        try {
+            Cli.main(new String[]{"-nsha256", "-dsha1", "-ftrue", temp.toString()});
+        } catch (final IllegalArgumentException exception) {
```

Comment:
@marceloamadeu One question, is `main()` being tested here supposed to throw an exception here, or not?

If it is, lets use `AssertThrows` instead. See https://stackoverflow.com/questions/40268446/junit-5-how-to-assert-an-exception-is-thrown. As it is now, if an exception is _not_ thrown the test will pass since the assertion is only in the catch block.

If not, let's remove the `catch` clause and declare `throws Exception` on this method so that it fails if an exception is thrown.